src/java.base/macosx/classes/sun/nio/fs/BsdNativeDispatcher.java

```diff
@@ -65,6 +65,61 @@ static native int fsstatEntry(long iter, UnixMountEntry entry)
     }
     static native byte[] getmntonname0(long pathAddress) throws UnixException;
 
+    /**
+     * ssize_t fgetxattr(int fd, const char *name, void *value, size_t size, u_int32_t position, int options);
+     */
+    static int fgetxattr(int fd, byte[] name, long valueAddress, int valueLen) throws UnixException
+    {
```

Comment:
Do you remind reformatting the changes to BsdNativeDispatcher.java to keep it consistent with the existing code and UnixNativeDispatcher?  Otherwise I think the patch is good and I can sponsor when you are ready.

Reply:
Tbh, I didn't find any official style guide (just for JavaFX). Should I hard wrap at 80 chars or what is the requirement here?

Reply:
Sorry, there isn't an agreed JDK wide style code. When it doubt, best to keep the style consistent with the existing code in the area. The main issue with BsdNativeDispatcher is that the line breaks are in unusual places.

Reply:
Oh if you're referring to the opening bracket being on a new line, that is already inconsistent in the Linux implementation, as well. If one tries to find a rule for this, it could be that the bracket should be on a new line, if the signature didn't fit into a single line.
In my latest commit I think I conformed to this style.